diff --git a/node_modules/detox/src/devices/common/drivers/android/tools/TempFileTransfer.js b/node_modules/detox/src/devices/common/drivers/android/tools/TempFileTransfer.js
index 42648fe..3815097 100644
--- a/node_modules/detox/src/devices/common/drivers/android/tools/TempFileTransfer.js
+++ b/node_modules/detox/src/devices/common/drivers/android/tools/TempFileTransfer.js
@@ -1,6 +1,6 @@
 const FileTransfer = require('./FileTransfer');
 
-const FILE_PATH = '/data/local/tmp/detox';
+const FILE_PATH = '~/tmp/detox';
 
 class TempFileTransfer extends FileTransfer {
   constructor(adb) {
